src/hatti/views/table.cljs

```diff
@@ -121,14 +120,19 @@
   ([form get-label? language & {:keys [is-filtered-dataview? owner]}]
    (let [columns (for [field (all-fields form :is-filtered-dataview?
                                          is-filtered-dataview?)]
-                   (let [{:keys [name type full-name]} field
+                   (let [{:keys [name type full-name], {:keys [hxl]} :instance} field
                          label (if get-label? (get-label field language) name)]
-                     {:id name :field full-name :type type
-                      :name label :toolTip label :sortable true
+                     {:id name
+                      :field full-name
+                      :type type
+                      :name (if hxl (str label "<div class=\"hxl-row\">" hxl "</div>") label)
```

Comment:
@pld Hm. How would alternate codings be accommodated in the future, if different styles are required for each? I think hxl-row works better here because of how specific it is. Is it being styled this way because it's a coding row, or because it's a _hxl_ coding row? That should be the answer.